HW5/Vaulin/home_work_5.1.py

```diff
@@ -0,0 +1,17 @@
+# Lv-677_Ivan_Vaulin
+
+# Task1. In the range from 1 to 10 determine 
+# Even numbers that are divisible by 2,
+# Odd numbers, which are divisible by 3,
+# Numbers that are not divisible by 2 and 3.
+
+x = []
```

Comment:
divisible_by_2 = []
divisible_by_3 = []
not_divisible_by_2_and_3 = []

for number in range(1,11):
    if number % 2 == 0:
        divisible_by_2.append(number)
    elif number % 3 == 0:
        divisible_by_3.append(number)
    else:
        not_divisible_by_2_and_3.append(number)